src/inspector/inspector_socket_server.cpp

```diff
@@ -338,9 +338,17 @@ bool InspectorSocketServer::HasTargets() {
 
 bool InspectorSocketServer::Start() {
   state_ = ServerState::kRunning;
+
   std::thread([this]() {

```

Comment:
>this [](http://example.com/codeflow?start=15&length=4)

So, what component ensures that this object is alive by the time lambda executes? Shouldn't it be a weak pointer?